### PR TITLE
Bug Fixes

### PR DESCRIPTION
**Pull request by simbazad**
# Description
Fixes several bugs

# Impacts
pfappserver && templates && network.pm

# Delete branch after merge

YES

### CODE REVIEW

File: html/pfappserver/lib/pfappserver/Form/Config/Network.pm [hunk 1]
```diff
@@ -70,7 +70,7 @@ has_field 'ip_assigned' =>
       option_pattern => sub {
           {
               message => "Must be a list of mac list ip address",
-              regex   => "^[0-9A-Fa-f][0-9A-Fa-f](:[0-9A-Fa-f][0-9A-Fa-f]){5}:\d{1,3}(.\d{1,3}){3}(,[0-9A-Fa-f][0-9A-Fa-f](:[0-9A-Fa-f][0-9A-Fa-f]){5}:\d{1,3}(.\d{1,3}){3})*\$",
+              regex   => "/^[0-9A-Fa-f][0-9A-Fa-f](:[0-9A-Fa-f][0-9A-Fa-f]){5}:\d{1,3}(.\d{1,3}){3}(,[0-9A-Fa-f][0-9A-Fa-f](:[0-9A-Fa-f][0-9A-Fa-f]){5}:\d{1,3}(.\d{1,3}){3})*\$/",
```
Comment (simbazad) [inline html/pfappserver/lib/pfappserver/Form/Config/Network.pm]: @jrouzierinverse  Regarding your previous note , i've added back the "" but also kept the / / in order to make it a valid regex and fix the -  "Unrecognized escape \d passed throug" ...

Comment (jrouzierinverse) [reply html/pfappserver/lib/pfappserver/Form/Config/Network.pm]: Now I understand why you wanted to change that good catch.
However the string should be.
```perl
"^[0-9A-Fa-f][0-9A-Fa-f](:[0-9A-Fa-f][0-9A-Fa-f]){5}:\\d{1,3}(.\\d{1,3}){3}(,[0-9A-Fa-f][0-9A-Fa-f](:[0-9A-Fa-f][0-9A-Fa-f]){5}:\\d{1,3}(.\\d{1,3}){3})*\$"
```
The \d was not escaped properly.

Comment (simbazad) [reply html/pfappserver/lib/pfappserver/Form/Config/Network.pm]: @jrouzierinverse Done

**Review by jrouzierinverse — CHANGES_REQUESTED** [review]
Fix the string then I will merge your request

**Pull request merged**